server/remote/gcal/get_default_calendar_view.go

```diff
@@ -83,8 +84,26 @@ func convertGCalEventToRemoteEvent(event *calendar.Event) *remote.Event {
 	start := convertGCalEventDateTimeToRemoteDateTime(event.Start)
 	end := convertGCalEventDateTimeToRemoteDateTime(event.End)
 
-	location := &remote.Location{
-		DisplayName: event.Location,
+	var conference *remote.Conference
+	var location *remote.Location
+
+	if event.ConferenceData != nil && len(event.ConferenceData.EntryPoints) > 0 {
+		conference = &remote.Conference{
+			URL: event.ConferenceData.EntryPoints[0].Uri,
+		}
+		if event.ConferenceData.ConferenceSolution != nil {
+			conference.Application = event.ConferenceData.ConferenceSolution.Name
+		}
+	} else if utils.IsURL(event.Location) {
+		conference = &remote.Conference{
+			URL: event.Location,
+		}
+	}
+
+	if !utils.IsURL(event.Location) {
+		location = &remote.Location{
```

Comment:
If I understand correctly, the output event's `event.Conference` and `event.Location` should never be defined at the same time? Just want to understand the implications of this change